tools/get-scala-commit-date

```diff
@@ -10,8 +10,13 @@
 
 [[ $# -eq 0 ]] || cd "$1"
 
-lastcommitdate=$(git log --format="%ci" HEAD | head -n 1 | cut -d ' ' -f 1)
-lastcommithours=$(git log --format="%ci" HEAD | head -n 1 | cut -d ' ' -f 2)
+if git rev-parse --is-inside-work-tree > /dev/null 2>&1; then
+	lastcommitdate=$(git log --format="%ci" HEAD | head -n 1 | cut -d ' ' -f 1)
```

Comment:
Yeah, the `head -n 1` part isn't very efficient.

As for TZ, that time ends up in the scala version number, so maybe it should be UTC or something, for the sake or consistency. I honestly don't know if it matters very much.